packages/api/src/routes/internal/ehr/patient.ts

```diff
@@ -120,36 +120,33 @@ router.get(
 );
 
 /**
- * POST /internal/ehr/:ehrId/patient/:id/resource/diff/set-entry-status
+ * POST /internal/ehr/:ehrId/patient/:id/resource/diff/:jobId/contribute
  *
- * Sets the status of a resource diff job entry.
+ * Contributes the resource diff bundle.
  *
  * @param req.query.ehrId - The EHR source.
  * @param req.query.cxId - The CX ID of the patient.
  * @param req.params.id - The patient id of the EHR patient.
- * @param req.query.jobId - The job ID.
- * @param req.query.entryStatus - The status of the entry.
+ * @param req.query.resourceType - The resource type to refresh.
+ * @param req.params.jobId - The job ID of the resource diff job.
  * @returns 200 OK
  */
 router.post(
-  "/:id/resource/diff/set-entry-status",
+  "/:id/resource/diff/:jobId/contribute",
```

Comment:
With the changes in endpoints, shouldn't we have diff releases to avoid breaking the flow, if we have lambdas or other ECS tasks that will call the old endpoint while the API gets switched to the new one?